metaflow/mflog/__init__.py

```diff
@@ -1,4 +1,11 @@
+import sys
 import math
+import time
+import select
+
+
+if (sys.version_info > (3, 0)):
+    from typing import Callable
```

Comment:
Given that we are not using types at the moment, can we dispense with this import?

Reply:
The types are used below in `delayed_update_while`. Even though it is using the py2-compatible comment syntax (instead of py3 typing annotations), the type checker still needs the import to work. Really helps if you e.g. use vscode and have type checking enabled.

Reply:
addressed in #504